packages/rum-core/src/domain/assembly.ts

```diff
@@ -98,6 +100,18 @@ export function startRumAssembly(
           ;(serverRumEvent.usr as Mutable<RumEvent['usr']>) = commonContext.user as User & Context
         }
         if (shouldSend(serverRumEvent, configuration.beforeSend, errorFilter)) {
+          if (!Number.isInteger(serverRumEvent.date)) {
```

Comment:
IE11 does not support `Number.isInteger`... maybe use something a bit more oldschool ?

```ts
typeof serverRumEvent.date === 'number' && serverRumEvent.date === Math.floor(serverRumEvent.date)
```

Reply:
Damn! nice catch 🙂

I think a typeof check should be enough here